can_tools/scrapers/official/NH/nh_demographics.py

```diff
@@ -0,0 +1,128 @@
+from tableauscraper import TableauScraper
+import us
+import pandas as pd
+from can_tools.scrapers import variables
+import multiprocessing
+from functools import partial
+
+from can_tools.scrapers.official.base import StateDashboard
+
+NUM_PROCESSES = 12
+
+
+class NHVaccineRace(StateDashboard):
+    has_location = False
+    source = "https://www.covid19.nh.gov/dashboard/vaccination"
+    source_name = "New Hampshire DHHS"
+    location_type = "county"
+
+    state_fips = us.states.lookup("New Hampshire").fips
+
+    variables = {
+        "Residents with At Least 1 Dose:": variables.INITIATING_VACCINATIONS_ALL,
+        "Residents Fully Vaccinated:": variables.FULLY_VACCINATED_ALL,
+    }
+
+    # hard coding to make parallel fetching easier
+    counties = [
+        "Rest of Hillsborough",
+        "Sullivan",
+        "Strafford",
+        "Rockingham",
+        "Nashua",
+        "Merrimack",
+        "Manchester",
+        "Grafton",
+        "Coos",
+        "Cheshire",
+        "Carroll",
+        "Belknap",
+    ]
+
+    def fetch(self):
+        # fetch NUM_PROCESSES counties at a time
+        pool = multiprocessing.Pool(processes=NUM_PROCESSES)
+        data = []
+
+        # Setting multiple parameter values on the same parameter type
+        # on a single TableauScraper instance removes the
+        # other selections (in this case, the county selection) from the instance, and there doesn't seem
+        # to be a good way to fix/circumvent this. So, we use two TableauScraper
+        # instances to fetch each respective dose type.
+        # This is an issue we've had in the past with the OH and WI demographic scrapers.
+        for variable in [
+            "Total Individuals with at least 1 Dose",
+            "Total Individuals Fully Vaccinated",
+        ]:
+            func = partial(self._get_county, variable)
+            data.extend(pool.map(func, self.counties))
+        return pd.concat(data)
+
+    @staticmethod
+    def _get_county(variable: str, county: str):
+        engine = TableauScraper()
+        engine.loads(
+            url="https://www.nh.gov/t/DHHS/views/VaccineOperationalDashboard/VaccineDashboard"
+        )
+
+        engine.getWorkbook().setParameter("ShowBy", "Race/Ethnicity")
+        engine.getWorkbook().setParameter("Metric", variable)
+
+        raw_data = []
+        worksheet = engine.getWorksheet("Count and Prop: Map (no R/E with town&rphn)")
+        workbook = worksheet.select("CMN + Town + RPHN", county)
+        raw_data.append(
+            workbook.getWorksheet("Count: Bar Chart").data.assign(location_name=county)
+        )
+
+        data = pd.concat(raw_data)
+        return data
+
+    def normalize(self, data) -> pd.DataFrame:
+        demo_replace = {
+            "Hispanic/Latino": "hispanic",
```

Comment:
Per slack, maybe move this to the ethnicity column?

Reply:
good call, just changed that 🙏